lib/ansible/galaxy/role.py

```diff
@@ -310,5 +310,3 @@ def spec(self):
         }
         """
         return dict(scm=self.scm, src=self.src, version=self.version, name=self.name)
-
-
```

Comment:
Not sure why there are a couple of empty lines at the bottom of the file either :). Probably don't need this though.